CU-CommunityApps/CD_cwd_events_localist_pull
# drupal 9 fix: config_export key

Drupal 9 compatibility fix:
* Add `config_export` key to `LocalistPull` config entity
* References:
  * [Drupal change record: define the `config_export` key in configuration entity annotation](https://www.drupal.org/node/2481909)
  * [Creating a configuration entity type > example/src/Entity/Example.php](https://www.drupal.org/docs/drupal-apis/configuration-api/creating-a-configuration-entity-type#s-examplesrcentityexamplephp)

----
### To test
...On a drupal 9 site -- or a drupal 8 site to ensure backwards compatibility:
1. Update your project with the `cet-fix` branch of this module:
    ```
    composer require cubear/cwd_events_localist_pull:dev-cet-fix
    ```
2. (Probably do a cache rebuild?? -- no db updates included in this change)
3. Save a change to your localist_pull entity.
    * Check 1: No WSOD upon save.
    * Check 2: Do config sync/export/import actions, make sure everything works right.
    * Check 3: Run cron, make sure import still works (**CAVEAT:** if you're testing on d9, your cron won't work until #22 is fixed.)
